Automattic/newspack-newsletters
# feat: tracking settings

### All Submissions:






### Changes proposed in this Pull Request:

1205078412187623-as-1205085335040733

Implements toggling the tracking features.

<img width="435" alt="image" src="https://github.com/Automattic/newspack-newsletters/assets/820752/756fcb61-5265-475a-adfd-bb8df9e4c618">

### How to test the changes in this Pull Request:

1. Check out this branch and visit Newsletters -> Tracking
2. Confirm you see the options toggled on by default
3. Create and send a newsletter with links
4. Confirm the links are proxied and the tracking pixel is also present
5. Visit the Tracking settings again, toggle both off and save
6. Visit the Newsletters table and confirm the "Opened" and "Clicks" columns don't render
7. Send another newsletter with links and confirm the URLs are not proxied and the pixel is not present

### Other information:
